public/wp-content/plugins/attendance-plugin/attendance.php

```diff
@@ -1,38 +1,267 @@
+											</tr>';
+				}
+				echo $table_allguests_body;
+			}
+			$table_allguests_footer.='</tbody>
+								</table>
+							</div>
+						</div>';
+			echo $table_allguests_footer;
+		}
+		wp_die();
+	}
+	add_action('wp_ajax_show_approved_guests','show_approved_guests');
+	add_action('wp_ajax_nopriv_show_approved_guests','show_approved_guests');            
+
+	function approve_guests(){
+		if(isset($_POST['approve_guest_id'])){
+			$guest_id=$_POST['approve_guest_id'];
+			update_post_meta($guest_id,'status','confirm');
+			wp_set_post_categories($guest_id,array(2));
+		}
+		wp_die();
+	}
+	add_action('wp_ajax_approve_guests','approve_guests');
+	add_action('wp_ajax_nopriv_approve_guests','approve_guests');          
+
+
+	function reject_guests(){
+		if(isset($_POST['reject_guest_id'])){
+			$guest_id=$_POST['reject_guest_id'];
+			var_dump($guest_id);
+			wp_set_post_categories($guest_id,array(3));
+		}
+		wp_die();
+	}
+	add_action('wp_ajax_reject_guests','reject_guests');
+	add_action('wp_ajax_nopriv_reject_guests','reject_guests');
+
+	function invited_guest_header()
+	{
+		$table_invitee_header='';
+		$table_invitee_header.='<div class="row">
+									<div class="col-lg-12 col-md-12 col-sm-12">
+										<div class="table-headings">Invited Guests</div><br>
+											<table class="table">
+												<thead>
+													<tr>
+														<th>Name</th>
+														<th>Email</th>
+														<th>Phone No.</th>
+														<th>Gender</th>
+														<th>Status</th>
+													</tr>
+												</thead>
+												<tbody>';
+		return $table_invitee_header;
+	}
+
+
+	function invited_guest_body($event_id)
+	{
+		$invited_guest_post_args=array(
+				'post_type' => 'guest',
+				'meta_key'=>'event_id',
+				'meta_value'=>$event_id,
+				'category_name'=>'approved'
+			);
+		$invited_guest_query = new WP_Query($invited_guest_post_args);
+		if ($invited_guest_query->have_posts()){    
+			$table_invitee_body=invited_guest_header();
+			while($invited_guest_query->have_posts()){
+				$invited_guest_query->the_post();
+				$phone=get_field('phone');
+				$name=get_the_title();
+				$email=get_field('email');
+				$status=get_field('status');
+				$gender=get_field('gender');
+				$table_invitee_body.=  '<tr class="table-success">
+											<td class="capitalize">'.$name.'</td>
+											<td>'.$email.'</td>
+											<td>'.$phone.'</td>
+											<td class="capitalize">'.$gender.'</td>';
+											if($status=='confirm'){
+												$table_invitee_body.='<td class="capitalize confirm-status"><span class="badge badge-success">'.$status.'</span></td></tr>';
+											}
+											else{
+												$table_invitee_body.='<td class="capitalize pending-status"><span class="badge badge-danger">'.$status.'</span></td></tr>';
+											}
+			}
+			$table_invitee_body.=invited_guest_footer();
+			return $table_invitee_body;
+		}			
+	}
+
+	function invited_guest_footer()
```

Comment:
I don't see any scope where this'll change. It's preferred to keep HTML in views rather than getting it from a function.